package.json

```diff
@@ -1,15 +1,15 @@
 {
-  "name": "react-template-typescript",
+  "name": "cgn-onboarding-portal-frontend",
   "version": "0.1.0",
   "license": "MIT",
   "scripts": {
-    "dev": "parcel --port 3000 src/index.html --out-dir build/debug",
+    "dev": "parcel --port 3000 src/index.html --public-url / --out-dir build/debug",
     "prebuild": "npm-run-all prebuild:*",
     "prebuild:clean": "shx rm -rf build",
     "prebuild:tsc": "tsc --noEmit",
-    "build": "parcel build src/index.html --out-dir build/release --public-url ./",
+    "build": "parcel build src/index.html --out-dir build --public-url /",
     "test": "jest",
-    "test:coverage": "jest --coverage",
+    "test:coverage": "jest --passWithNoTests --coverage --coverageReporters=cobertura",
```

Comment:
Why this option `--passWithNoTests` ?

Reply:
For testing the next Publish task in the pipeline because there are no tests yet